pykechain/models/property2_multi_reference.py

```diff
@@ -194,7 +119,7 @@ def set_prefilters(
         ))
 
         options_to_set = self._options
-        options_to_set['prefilters'] = {'property_value': ','.join(list_of_prefilters) if list_of_prefilters else {}}
+        options_to_set['prefilters'] = {'property_value': ','.join(list_of_prefilters)} if list_of_prefilters else {}
```

Comment:
Is this correct, can you re-evaluate? Is this what we want that the prefilters is set to {} when no prefilters are there instead of before `prefilters= {'property_value': {}}`

Reply:
Good point. In hindsight, having this dict structure (albeit empty) would be more robust if it is retrieved anywhere else.

I plan to simply clear the "prefilters" from the options entirely (if overwrite is true) or do nothing at all (if overwrite is false), saving 1 edit request.

Note, the "property_value" is a string field (see the `','.join` method) and not a dict. Therefore, i'll replace the original `else {}` with `else ""` nonetheless.